Client/Frontend/Strings.swift

```diff
@@ -875,13 +880,14 @@ extension String {
         public static let IntroWelcomeTitle = MZLocalizedString(
             "Onboarding.Welcome.Title.v106",
             tableName: nil,
-            value: "Welcome to an independent internet",
+            value: "Make Firefox your go-to browser",
```

Comment:
If the value for a string is changed, we need to modify the key as well of the string otherwise the export/import won't work.